src/MeetingsSDKAdapter.js

```diff
@@ -180,30 +189,59 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
   }
 
   /**
-   * Returns a promise to a local media stream based on the given constraints.
+   * Returns a promise to a local media stream based on the given mediaDirection, audioVideo options.
    *
    * @see {@link MediaStream|https://developer.mozilla.org/en-US/docs/Web/API/MediaStream}.
    *
    * @private
    * @param {string} ID ID of the meeting for which to fetch streams
-   * @param {object} constraint Media stream constraints
+   * @param {object} mediaDirection A configurable options object for joining a meetings
```

Comment:
maybe switch to double spaces for this whole block? if you prefer not to change it's ok

Reply:
on sdk adapter we didn't add double spaces between the param name and description, all the others have just one space